src/views/admin/PersonDetailsView.vue

```diff
@@ -113,6 +118,44 @@
     await personStore.deletePersonById(personId);
   }
 
+  function keyMapper(key: string): string {
+    switch (key) {
+      case 'lock_locked_from':
+        return t('person.lockedBy');
+      case 'lock_timestamp':
+        return t('person.lockedSince');
+      default:
+        return key;
+    }
+  }
+
+  function keyValueMapper(key: string, value: string): string {
+    switch (key) {
+      case 'lock_timestamp':
+        return new Intl.DateTimeFormat('de-DE', {
+          year: 'numeric',
+          month: '2-digit',
+          day: '2-digit',
+        }).format(new Date(value));
+      default:
+        return value;
+    }
+  }
```

Comment:
Ein Switch case statement mit nur 1 Case ist überflüssig:

```suggestion
function keyValueMapper(key: string, value: string): string {
  if (key === PersonKey.LockTimestamp) {
    return new Intl.DateTimeFormat('de-DE', {
      year: 'numeric',
      month: '2-digit',
      day: '2-digit',
    }).format(new Date(value));
  }
  return value;
}
```